src/components/table/mixins/sort.js

```diff
@@ -0,0 +1,62 @@
+import { EVENT_THEAD_CLICKED } from "../helpers/constants"
+import "./sort.scss"
+
+const SORT_DIRECTION_ASC = "asc"
+const SORT_DIRECTION_DESC = "desc"
+
+export default {
+  data() {
+    return {
+      sortBy: null,
+      sortDir: SORT_DIRECTION_ASC,
+    }
+  },
+  created() {
+    if (this.isSortable) {
+      this.$on(EVENT_THEAD_CLICKED, this.handleSort)
+    }
+  },
+  computed: {
+    sortedData() {
+      const data = [...this.data]
```

Comment:
You could have an early return `this.data` if `this.sortBy === null`

Reply:
fixed, good catch. now tableData makes more sense also for the moment